README.md

```diff
@@ -1 +1,178 @@
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML & CSS
+- Linters
+- Git & Github
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Git in your local machine
+- Code editor (VS code, Atom, etc)
+- Terminal
+
+### Setup
+
+- git clone git@github.com:Kel-nana/Hello.git
+- cd Hello
+
+### Usage
+
+- Add the changes you want and feel free to use in your project!
+
+### Run tests
+
+- Check that all the tests are correctly passed🤝
+
+### Merge
+
+- Merge it so we can all see your work!!🤝
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@Kel-nana](https://github.com/Kel-nana)
+- Linkedin: [@kelvin-irungu](https://www.linkedin.com/in/kelvin-irungu-838923249/)
+  <<<<<<< HEAD
+- Twitter: [@kelvin-irungu](https://www.linkedin.com/in/kelvin-irungu-838923249/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://twitter.com/home?lang=en).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Share ideas on how to improve this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+# <<<<<<< HEAD
+
+> > > > > > > 71fd8824e4b286731a36c47fd8c04dbd5ab77f65
+> > > > > > > I would like to thank Microverse documentation and my team mates for all there resources and support.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+# <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse documentation and Myself ⭐️
```

Comment:
- [ ] Good job for using the new [README template](https://github.com/microverseinc/readme-template), Kindly I recommend you to customize the README file according to the current project, you used the "Hello-Micro verse" Project and make sure to list out the "Key-feature" and "Future Features", including 1-3 potential features to be added.